azure-pipelines/components/azure-pipeline-public-api.yml

```diff
@@ -85,7 +85,7 @@ variables:
   - name: test
     value: ${{ or(parameters.test, contains(variables['Build.SourceBranch'],'feature1')) }}
   - name: pp
-    value: ${{ and(or(parameters.pp, contains(variables['Build.SourceBranch'],'main')), eq(variables.buildNotNeeded, 'false')) }}
+    value: ${{ and(or(parameters.pp, contains(variables['Build.SourceBranch'],'main1')), eq(variables.buildNotNeeded, 'false')) }}
```

Comment:
Why is `feature1` and `main1` being used here?

Reply:
Hello @kruncher answer below.
To disable auto-triggering, instead of turning off the pipeline in Azure DevOps, we’re modifying the value that activates the trigger. Currently, the apps are in the Tran environment, so auto-triggering for other environments isn’t necessary. When we start moving them, we'll update the trigger settings to use the correct value.

Reply:
So effectively this is the same as commenting out the "pp" aspect?